src/twilio/assets/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
do we need these html and css assets? it doesn't look like they're served anywhere

couldn't / shouldn't we store assets like pause1.mp3 here?

Reply:
ya the html + css in there are just the defaults i think, i will delete them!

and pause1.mp3 is stored in src/twilio/assets ... it has a path that doesn't show /assets, but that's where the file is!